src/lib/nlcManager.js

```diff
@@ -29,6 +33,7 @@ function NLCManager(options) {
 	this.opts.classifierName = options.classifierName || 'default-classifier';
 	this.opts.maxClassifiers = options.maxClassifiers || 3;
 	this.opts.classifierLanguage = options.language || 'en';
+	this.opts.saveTrainingData = options.saveTrainingData || true;
```

Comment:
Changed to 
`this.opts.saveTrainingData = options.saveTrainingData === false ? options.saveTrainingData : true;`

Lint complained of a simpler syntax
`this.opts.saveTrainingData = options.saveTrainingData === false ? false : true;`